.github/actions/build_setup/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Build Setup"
+description: "Set up building environment"
+inputs:
+  github_token:
+    description: "Github Token"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Configure Cargo
+      shell: bash
+      run: |
+        cp ./.cargo/ci.toml $HOME/.cargo/config.toml
+
+    - name: Ubuntu install clang and library
+      if: ${{matrix.config.os == 'ubuntu-latest'}}
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y clang clang-tools libclang-dev llvm
+
+    - name: Set up mold v1.1
+      if: ${{matrix.config.os == 'ubuntu-latest'}}
+      shell: bash
+      run: |
+        MOLD_LINK="https://github.com/rui314/mold/releases/download/v1.1/mold-1.1-x86_64-linux.tar.gz"
```

Comment:
Since we only use `MOLD_LINK` once, how about using it directly?

```shell
wget -O mold.tar.gz https://github.com/rui314/mold/releases/download/v1.1/mold-1.1-x86_64-linux.tar.gz
```

Or, maybe we can set `MOLD_VERSION=1.1` here?